net-dialup/linux-atm/linux-atm-2.5.2-r2.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools flag-o-matic linux-info
+
+DESCRIPTION="Tools for ATM"
+HOMEPAGE="https://linux-atm.sourceforge.net/"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+BDEPEND="sys-devel/bison"
+
+RESTRICT="test"
+
+CONFIG_CHECK="~ATM"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-headers.patch
+	"${FILESDIR}"/${P}-linux-5.2-SIOCGSTAMP.patch
+	"${FILESDIR}"/${P}-linux-headers-5.19.patch
+	"${FILESDIR}"/${P}-c99-musl.patch
+)
+
+src_prepare() {
+	default
+
+	sed -i '/#define _LINUX_NETDEVICE_H/d' \
+		src/arpd/*.c || die "sed command on arpd/*.c files failed"
+
+	eautoreconf
+}
+
+src_configure() {
+	append-flags -fno-strict-aliasing
+	export YACC=bison
```

Comment:
`YACC` as `bison` is unusual. It should usually be `bison -y`...

Reply:
It doesn't break compile! Not sure about resulting code